content/_help/verify-your-identity/overview._zh.md

```diff
@@ -1,19 +1,51 @@
 ---
-layout: help
 title: 验证你的身份
 category: verify-your-identity
 permalink: /zh/help/verify-your-identity/overview/
 meta_title: 概述
 order: 0
+redirect_from:
+  - /zh/help/verify-your-identity/
+  - /zh/help/identity-verification/i-have-more-than-one-logingov-account-can-I-verify-my-identity-for-all-of-them/
+  - /zh/help/identity-verification/related-articles/
+  - /zh/help/identity-verification/why-do-i-need-to-verify-my-identity/
+  - /zh/help/verifying-your-identity/multiple-accounts-to-verifying-my-identity-for/
+  - /zh/help/verifying-your-identity/why-do-i-need-to-verify-my-identity/
```

Comment:
IIRC these were legacy articles for other languages which never existed in Chinese, which I'd assume is why they didn't exist here previously.

Or is adding them for consistency's sake?

Reply:
Ahh, you are right. Probably I'll just remove them to reduce confusion, since it's unlikely a user would actually go to those urls.